app/models/apple/publisher.rb

```diff
@@ -114,6 +114,10 @@ def wait_for_upload_processing(eps)
         pdfs = eps.map(&:podcast_delivery_files).flatten
 
         Apple::PodcastDeliveryFile.wait_for_delivery_files(api, pdfs)
+
+        # Get the latest state of the podcast containers
+        # which should include synced files
+        Apple::PodcastContainer.poll_podcast_container_state(api, eps)
```

Comment:
Again, missing local state, this time in the `files` Apple attribute -- which gives a readout of the successfully processed/delivered files.